packages/tools/src/commands/integrity/files.ts

```diff
@@ -172,7 +244,7 @@ const verifyCodeowners = (plugins: Workspace[]) => {
 
 export const updateFiles = async (plugins: Workspace[]) => {
     const errors: string[] = [];
-    await updateFactory(plugins);
+    errors.push(...(await updateFactory(plugins)));
```

Comment:
Is there some reason that prevents doing this as a one liner?

```ts
const errors: string[] = await updateFactory(plugins);
```

Reply:
Absolutely no reason at all, apart from some refactoring residue.
I'll update.